src/py/aspen/config/config.py

```diff
@@ -233,9 +233,16 @@ def DATABASE_URI(self) -> str:
         username = self.AWS_SECRET["DB"]["rw_username"]
         password = self.AWS_SECRET["DB"]["rw_password"]
 
-        rds = boto3.client("rds")
-        response = rds.describe_db_instances(DBInstanceIdentifier="aspen-db")
-        instance_info = response["DBInstances"][0]
-        instance_address = instance_info["Endpoint"]["Address"]
-        instance_port = instance_info["Endpoint"]["Port"]
-        return f"postgresql://{username}:{password}@{instance_address}:{instance_port}/aspen_db"
+        instance_address = None
+        try:
+            instance_address = self.AWS_SECRET["DB"]["address"]
```

Comment:
There's a new AWS secret for this environment (rdev/aspen_config). The prefix lets us have a different secret for remote dev / staging / prod